include/glow/Backends/CompiledFunction.h

```diff
@@ -60,11 +61,25 @@ class CompiledFunction {
   /// Collects constants for runtime.
   virtual void collectConstants(Module *){};
 
+  /// Setter for TraceEvent lookup. Note: does not enable tracing automatically.
```

Comment:
Why is this a property of the compiled function?

Reply:
Because it seems to me that which Trace Events are included are a property of the function. TraceInfo is the static per-function part of the profiling data, not the metrics themselves but how to interpret them (e.g. the first timestamp is the start of the function, the second timestamp is the end of Conv1 and the start of RELU, etc etc).

Storing them outside the compiled function means in the Runtime we'd need to keep a map in the Executor (?) of run -> TraceInfo and do interpretation there. It would add dependencies on the backend to the Executor, but would reduce latency.